src/sequelize/sequelize/sequelize.ts

```diff
@@ -84,11 +85,13 @@ export class Sequelize extends OriginSequelize {
     return models.map(model => {
       const modelName = getModelName(model.prototype);
       const attributes = getAttributes(model.prototype);
+      const indexes = getIndexes(model.prototype);
       const modelOptions = getOptions(model.prototype);
 
       if (!modelOptions) throw new Error(`@Table annotation is missing on class "${model['name']}"`);
 
       const initOptions: InitOptions & { modelName } = {
+        indexes: indexes && Object.values(indexes),
```

Comment:
Can you use `es6` compatible features (`Object.keys(indexes).map(key => indexes[key])`) instead? This wasn't clear from tsconfig.json. Since there is `lib` set to `["es2017"]`, which is wrong. I'm going to fix that. Sorry for that.

Reply:
Okay, no problem.